src/collections/Breadcrumb/BreadcrumbSection.js

```diff
@@ -93,4 +93,4 @@ export default class BreadcrumbSection extends Component {
   }
 }
 
-BreadcrumbSection.create = createShorthandFactory(BreadcrumbSection, content => ({ content, link: true }))
+BreadcrumbSection.create = createShorthandFactory(BreadcrumbSection, content => ({ content, link: true }), true)
```

Comment:
This is how we now flag a factory to `generateKey`.  It will generate a key as a hash of shallow props.